framework/web/User.php

```diff
@@ -562,6 +545,48 @@ protected function sendIdentityCookie($identity, $duration)
     }
 
     /**
+     * Determines if an identity cookie has a valid format and contains a valid auth key. 
+     * This method is used when [[enableAutoLogin]] is true.
+     * This method attempts to authenticate a user using the information in the identity cookie.
+     * @return array|null Returns an array of 'identity' and 'duration' if valid, otherwise null.
```

Comment:
Based on what it returns I'd name a method something like `getIdentityAndDurationFromCookie()`. Rather long but probably reflects what's done inside better.